posts/2025-12-02-project-goals-nov-update.md

```diff
@@ -0,0 +1,239 @@
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/117'><strong>Rust 2024 Edition</strong></a></div>
+    <div style="flex: initial;"><progress value="22" max="30"></progress>
+</div>
+</div>
+
+Rust 2024 has now entered the nightly beta and is expected to stabilize as part of Rust 1.85 on 2025-02-20.  It has a great many improvements that make the language more consistent and ergonomic, that further upon our relentless commitment to safety, and that will open the door to long-awaited features such as gen blocks, let chains, and the never type `!`. For more on the changes, see the [nightly Edition Guide.](https://doc.rust-lang.org/nightly/edition-guide/rust-2024/index.html) The [call for testing blog post](https://blog.rust-lang.org/2024/11/27/Rust-2024-public-testing.html) contains more information and instructions on how you can try it yourself.
+
+## Goals with updates
+
+<div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/100'><strong>&quot;Stabilizable&quot; prototype for expanded const generics</strong></a></div>
+    <div style="flex: initial;"><progress value="0" max="4"></progress>
+</div>
+</div>
+
+ * `min_generic_const_args` now exists as a feature gate, though without any functionality, only some gated refactorings, but shouldn't be long before it has actual functionality behind it.
+* The refactoring to remove all the `eval_x` methods on `ty::Const` has been completed, making it possible to correctly implement normalization for constants.
+
+
+<div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/102'><strong>Assemble project goal slate</strong></a></div>
+    <div style="flex: initial;"><progress value="6" max="8"></progress>
+</div>
+</div>
+
+* Posted the October update.
+* Created more automated infrastructure to prepare the October update, making use of an LLM to summarize updates into one or two sentences for a concise table.
+
+
+<div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/104'><strong>Begin resolving &#x60;cargo-semver-checks&#x60; blockers for merging into cargo</strong></a></div>
+    <div style="flex: initial;"><progress value="0" max="4"></progress>
+</div>
+</div>
+
+* Support for cargo manifest linting is [now merged](https://github.com/obi1kenobi/cargo-semver-checks/pull/1007), making it possible to catch breakage caused by _manifest_ (`Cargo.toml`) changes, not just source code changes. An example of such breakage is the removal of a package feature: any crates that enabled the removed feature will no longer build.
+* Partial schema design and implementation of type information in lints, enabling the creation of breaking-change lints and improving diagnostic quality for type-related breaking changes.
```

Comment:
```suggestion
* Partial schema design and implementation of type information in lints, enabling the creation of breaking-change lints and improving diagnostic quality for a subset of type-related breaking changes.
```
Just to avoid the possibility of misconceptions. The full suite of type-related breakage lints is not yet possible, and work on that is ongoing.